JSIL.Libraries/Includes/Bootstrap/Resources/Main.js

```diff
@@ -8,7 +8,7 @@ if (!$jsilcore)
 
 $jsilcore.getCurrentUICultureImpl = function () {
   var language;
-  var svc = JSIL.Host.getService("window", false);
+  var svc = JSIL.Host.getService("window", true);
```

Comment:
Ah, I probably got the second argument's boolean nature backwards... bools are awful.